pocs/tests/test_observatory.py

```diff
@@ -379,3 +379,18 @@ def test_operate_dome(config_with_simulated_dome):
     assert observatory.open_dome()
     assert observatory.dome.is_open
     assert not observatory.dome.is_closed
+
+
+def test_create_flat_field(observatory):
```

Comment:
This implies that most of the new code is untested (i.e. no automated test, not that you haven't tested it on PAN001).

Reply:
Yes. I had not marked this for review yet because I haven't written the tests. I just wrote some to cover the creation of the flat-field `Observation` but haven't yet figured out a good way to do the actual `take_flat_field` method. So far this PR is just extracting from #541. I'll mark as WIP until I think it is ready but as mentioned I might just end up with an alternative PR.